lib/utils/colors.js

```diff
@@ -79,6 +79,16 @@ module.exports = new (function() {
     });
   };
 
+  const enable = function() {
+    Settings.enabled = true;
+
+    Object.keys(ConsoleColor.prototype).forEach(function (color) {
```

Comment:
this doesn't look like it should be here.

Reply:
not needed, removed.